procedures/release-duty/index.rst

```diff
@@ -199,48 +200,43 @@ Firefox bookmarks
 
 These bookmarklets should help you view tasks and taskgroups in Firefox.
 
-*  Go to Bookmarks -> Show All Bookmarks
-*  Gear symbol -> New Bookmark
-*  Name: ``task inspector`` Location: `https://tools.taskcluster.net/tasks/%s <https://tools.taskcluster.net/tasks/%s>`__ ; Keyword: ``task``
-*  Name: ``taskgroup inspector`` Location: `https://tools.taskcluster.net/groups/%s <https://tools.taskcluster.net/groups/%s>`__ ; Keyword: ``taskgroup``
-*  Name: ``stop`` Location: ``javascript:stop();``
-   *  This can be used to stop further loading in the Task Group Inspector. It shouldn't be used when actively monitoring (i.e.: watching for failures), but it can greatly speed things up if you're using it for other reasons. Be sure to wait for the initial tasks to load before you use it.
+ * Go to Bookmarks -> Show All Bookmarks
+ * Gear symbol -> New Bookmark
+ * Name: ``task inspector`` Location: `https://tools.taskcluster.net/tasks/%s <https://tools.taskcluster.net/tasks/%s>`__ ; Keyword: ``task``
```

Comment:
The TC UI changed and we use firefox-ci-tc.services.mozilla.com instead of the defunct taskcluster instance. We could remove these docs or amend them, either works. The new UI is smoother so we could do former too.